src/coreclr/inc/patchpointinfo.h

```diff
@@ -157,12 +157,12 @@ struct PatchpointInfo
     // FP relative offset of this local in the original method
     int Offset(unsigned localNum) const
     {
-        return (m_offsetAndExposureData[localNum] & ~EXPOSURE_MASK);
+        return (m_offsetAndExposureData[localNum] >> OFFSET_SHIFT);
     }
 
     void SetOffset(unsigned localNum, int offset)
     {
-        m_offsetAndExposureData[localNum] = offset;
+        m_offsetAndExposureData[localNum] = (offset << OFFSET_SHIFT);
```

Comment:
Seems like this will clear out the exposure bit.

Reply:
It's always been that way and calling `SetIsExposed` was optional. I'll rework this so they get set together.